### PR TITLE
[Add] auth pages web ui

### PR DESCRIPTION
**Pull request by siva2204**

### CODE REVIEW

File: lib/pages/auth/check_mail_page.dart [hunk 1]
```diff
@@ -7,39 +13,70 @@ class CheckMailPage extends StatelessWidget {
...
+            child: (() {
+              var screenwidth = MediaQuery.of(context).size.width;
+
+              if (screenwidth > 1000) {
+                return Container(
+                  decoration: BoxDecoration(
+                      border: Border.all(color: secondaryColor, width: 2),
+                      borderRadius:
+                          const BorderRadius.all(Radius.circular(10))),
+                  // decoration: BoxDecoration(),
```
Comment (dhruv1294) [inline lib/pages/auth/check_mail_page.dart:35]: remove this comment

File: lib/pages/auth/forgot_password_page.dart [hunk 2]
```diff
@@ -31,26 +33,48 @@ class ForgotPasswordPage extends StatelessWidget {
...
               }
-              return body();
+
+              var screenwidth = MediaQuery.of(context).size.width;
+
+              return screenwidth > 1000
+                  ? (Center(
+                      child: Container(
+                      decoration: BoxDecoration(
+                          // color: baseColor,
```
Comment (dhruv1294) [inline lib/pages/auth/forgot_password_page.dart:45]: here as well

File: lib/pages/auth/forgot_password_page.dart [hunk 3]
```diff
@@ -31,26 +33,48 @@ class ForgotPasswordPage extends StatelessWidget {
...
+                          border: Border.all(color: secondaryColor, width: 2),
+                          borderRadius:
+                              const BorderRadius.all(Radius.circular(10))),
+                      // decoration: BoxDecoration(),
```
Comment (dhruv1294) [inline lib/pages/auth/forgot_password_page.dart:49]: here too

File: lib/pages/auth/verify_phone/enter_otp_page.dart [hunk 4]
```diff
@@ -35,9 +37,26 @@ class _EnterOtpPageState extends State<EnterOtpPage> {
...
+                var screenwidth = MediaQuery.of(context).size.width;
+
+                return screenwidth > 1000
+                    ? (Center(
+                        child: Container(
+                        decoration: BoxDecoration(
+                            border: Border.all(color: secondaryColor, width: 2),
+                            borderRadius:
+                                const BorderRadius.all(Radius.circular(10))),
+                        // decoration: BoxDecoration(),
```
Comment (dhruv1294) [inline lib/pages/auth/verify_phone/enter_otp_page.dart:49]: here too

File: lib/pages/landing_page.dart [hunk 5]
```diff
@@ -12,63 +13,94 @@ class LandingPage extends StatelessWidget {
...
-                    height: 300,
+        body: FillMaxHeightScrollView(
+          builder: (context) => SafeArea(
+            child: (() {
+              var screenwidth = MediaQuery.of(context).size.width;
+
+              return screenwidth > 1000
+                  ? Container(
+                      decoration: BoxDecoration(
+                          // color: baseColor,
```
Comment (dhruv1294) [inline lib/pages/landing_page.dart:24]: here as well

File: lib/utils/form_validation.dart [hunk 6]
```diff
@@ -0,0 +1,30 @@
+import 'package:reactive_forms/reactive_forms.dart';
```
Comment (Sudhindra3000) [inline lib/utils/form_validation.dart:1]: Rename file to `validation_messages.dart`

**Review by Sudhindra3000 — CHANGES_REQUESTED** [review]

**Review by Sudhindra3000 — APPROVED** [review]

**Review by dhruv1294 — APPROVED** [review]

**Pull request merged**